deployments/helm/hephaestus/templates/crd-hooks.yaml

```diff
@@ -93,9 +94,6 @@ spec:
           imagePullPolicy: {{ .Values.controller.manager.image.pullPolicy }}
           args:
             - crd-apply
-            {{- if .Values.istio.enabled }}
-            - --istio-enabled
```

Comment:
should we remove support for this flag as a follow-up?

Reply:
either remove it or fix it.
it doesn't handle the case where istio is enabled and there is no sidecar. Basically it shouldn't care and just deal with the case when it does find a sidecar.

Reply:
realize this is broken in all places that copied this code, ie, 2 other locations.